components/ScoreLabel.js

```diff
@@ -0,0 +1,18 @@
+import { Text } from "react-native";
+
+export const ScoreLabel = ({ currentPoints }) => {
+  return (
+    <Text
+      style={{
+        textAlign: "center",
+        color: "white",
+        fontSize: 50,
+        fontWeight: "bold",
+        margin: 20,
+        zIndex: 999,
```

Comment:
@leahkatherine added a zindex here to make it render above everything else, all the time. Keep this in mind if you wanted to move the score label around depending on your game state, as this can also be achieved by organization in your 'view' tree.